lib/zfs.js

```diff
@@ -140,13 +151,41 @@ zpool.status = function (pool, callback) {
  * zpool create command, including log devices, cache devices, and hot spares.
  * The input is an object of the form produced by the disklayout library.
  */
-zpool.create = function (pool, config, callback) {
+zpool.create = function (pool, config, force, encrypt, efi, callback) {
+	var cmd;
 	var args;
 
-	if (arguments.length != 3)
-		throw Error('Invalid arguments, 3 arguments required');
+	if (arguments.length === 3) {
+		callback = force;
+		encrypt = false;
+		force = false;
+		efi = true;
```

Comment:
For the linux-cn branch, we may need to set this to false as I'm not sure zpool has a -B on Linux.  OTOH if we only ever access this through mkzpool, we explicitly set "efi" from the caller, and can therefore DTRT.

Reply:
I'll do an `if` clause for Linux.